config/environments/development.rb

```diff
@@ -97,3 +97,9 @@
   end
 
 end
+
+
+ActiveMerchant::Billing::Base.mode = :test
+
+  ::GATEWAY = ActiveMerchant::Billing::StripeGateway.new(
+    :login => ENV['SECRET_KEY'])
```

Comment:
well, as the `SECRET_KEY` is the gateway API key, I think that using environment is the most secure way of providing it into the application... please suggest if I should change this or not... but I think I can definitely make the name more clear to something like `PAYMENT_GAYEWAY_SECRET_KEY`